ipc/src/server.rs

```diff
@@ -332,13 +339,15 @@ mod tests {
 
```

Comment:
This test still occasionally fails here. Running `for i in {1..20}; do cargo test; done` it fails 2-3 times. Running it on by itself (`for i in {1..20}; do cargo test req_parallel; done`) makes things worse: about a fourth fails.